WordPress/gutenberg
# Adapt Gutenberg code to work after global styles lands in core

Companion for https://github.com/WordPress/wordpress-develop/pull/1262

Adapts Gutenberg code to work once we shipped "global styles" to WordPress core.

## Testing scenarios

- with WordPress 5.7.2 and lower
- with WordPress 5.8-alpha
- with WordPress 5.8-alpha and this https://github.com/WordPress/wordpress-develop/pull/1262

Instructions:

- Load a theme with theme.json support (TT1-blocks) and make sure it looks like it's supposed to. A good litmus test is that colors show properly and the color palette shown in the color UI controls is the one by the theme.
- Load a theme without theme.json support (TT1) and do the same.
